apps/champions/lib/champions/battle/simulator.ex

```diff
@@ -195,6 +195,16 @@ defmodule Champions.Battle.Simulator do
               },
               :skill_action
             )
+            |> add_to_history(
+              %{
+                target_id: unit.id,
+                stat_affected: %{
+                  stat: :ENERGY,
+                  amount: -@ultimate_energy_cost
```

Comment:
This should be 0, otehrwise the whole concept of stat_override doesn't make sense